.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+/log/
+*.log
+.DS_Store
+.eslintrc.js
+
+package-lock.json
+package.json
```

Comment:
These files should not be excluded.
Basically, this is the first point in the issue: `create a new npm project`.
Please read the following to get to know more about `package.json` and `package-lock.json`
- https://nodejs.dev/learn/an-introduction-to-the-npm-package-manager
- https://nodejs.dev/learn/the-package-json-guide
- https://nodejs.dev/learn/the-package-lock-json-file

Reply:
Tks for review. I`ll back it to head directory.